joern-cli/frontends/c2cpg/src/test/scala/io/joern/c2cpg/cpp/features17/Cpp17FeaturesTests.scala

```diff
@@ -281,24 +345,26 @@ class Cpp17FeaturesTests extends AstC2CpgSuite(fileSuffix = FileDefaults.CppExt)
           |}
           |""".stripMargin)
```

Comment:
I'm missing the constructor call for `std::unordered_map<std::string, int> mapping` in the calls below. Is that because the definition for that type is missing in this test?

Reply:
That one was missing because the type can't be resolved. I worked around that and now its there.
(as always: "problem bindings" need to be handeled manually)